api/cmd/main.go

```diff
@@ -48,14 +49,13 @@ func main() {
 	if flag == "s3" {
 		s3ws := new(restful.WebService)
 		s3ws.Path("/")
-		s3ws.Doc("OpenSDS Multi-Cloud API")
+		s3ws.Doc("OpenSDS Multi-Cloud S3 API")
```

Comment:
Suggest to kee 'S3' here, this is only for S3 APIs, it's URL, authentication, etc. is different from other multi-cloud services, and be compatible with aws s3.

Reply:
U mean keep 'S3' right? I have added 'S3' not removed